charactersheet/charactersheet/viewmodels/dm/encounter.js

```diff
@@ -14,7 +14,8 @@ function EncounterViewModel() {
 
     self.sections = [
         { property: 'pointOfInterestSectionViewModel', vm: PointOfInterestSectionViewModel, model: PointOfInterestSection },
-        { property: 'notesSectionViewModel', vm: NotesSectionViewModel, model: NotesSection }
+        { property: 'notesSectionViewModel', vm: NotesSectionViewModel, model: NotesSection },
+        { property: 'npcSectionViewModel', vm: NPCSectionViewModel, model: NPCSection }
```

Comment:
This order determines the order that items appear in the Encounter Detail VM. Notes should be last. Can we move these around? I'd actually prefer that NPC goes above POI.